dpgen2/exploration/task/lmp_spin_task_group.py

```
@@ -0,0 +1,126 @@
+        self,
+    ):
+        super().__init__()
+        self.lmp_set = False
+        self.plm_set = False
+
+    def set_lmp(
+        self,
+        numb_models: int,
+        lmp_template_fname: str,
+        plm_template_fname: Optional[str] = None,
+        revisions: dict = {},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)